pyproject.toml

```
@@ -53,11 +54,12 @@ lint = "pylint protostar"
 local_static_check = ["format", "lint", "type_check", "custom_checks"]
 profile = "python -m cProfile -o recent_profiling.prof protostar.py"
 show_prof = "snakeviz recent_profiling.prof"
-test = ["test_unit", "test_integration", "build", "test_e2e"]
+test = ["test_unit", "test_integration", "build", "test_e2e", "test_performance"]
```

```suggestion
test = ["test_unit", "test_integration", "test_performance", "build", "test_e2e"]
```
I'd move performance tests before "build" to make the feedback loop faster.